### ISSUE TITLE
getting error while running train.py file

### ISSUE THREAD

**Issue by neelima1719** [sagieppel/Fully-convolutional-neural-network-FCN-for-semantic-segmentation-Tensorflow-implementation#5]
I am getting following error 

 from ._conv import register_converters as _register_converters
npy file loaded
build model started
WARNING:tensorflow:From /home/dtrl/Downloads/FCN/Fully-convolutional-neural-network-FCN-for-semantic-segmentation-Tensorflow-implementation-master/BuildNetVgg16.py:110: calling argmax (from tensorflow.python.ops.math_ops) with dimension is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
FCN model built
Setting up Saver...

Kernel died, restarting

**Comment by sagieppel** [comment]
All these messages are standard so I cant deduce much from it.
In which line of the code the program crashed?

**Comment by neelima1719** [comment]
The code basically giving error here:

Net =  BuildNetVgg16.BUILD_NET_VGG16(vgg16_npy_path=model_path) #Create
class for the network
    Net.build(image, NUM_CLASSES,keep_prob)# Create the net and load intial
weights
#......................................Get loss functions for neural net
work  one loss f

Thanks and Regards
Neelima Chauhan
SRF
DTRL, DRDO
New Delhi

**Comment by neelima1719** [comment]
Sorry for the prvs reply but now it is coming in this code:
 print("Setting up Saver...")
    saver = tf.train.Saver()
    sess.run(tf.global_variables_initializer()) #Initialize variables
   * ckpt = tf.train.get_checkpoint_state(logs_dir)*
*    if ckpt and ckpt.model_checkpoint_path: # if train model exist restore
it*
*        saver.restore(sess, ckpt.model_checkpoint_path)*
        print("Model restored...")

Thanks and Regards
Neelima Chauhan
SRF
DTRL, DRDO
New Delhi

**Comment by sagieppel** [comment]
Not sure I understand the program crush in the line? 
saver.restore(sess, ckpt.model_checkpoint_path)
